src/mon/Monitor.cc

```diff
@@ -2631,7 +2631,19 @@ void Monitor::handle_command(MonOpRequestRef op)
     return;
   }
 
-  cmd_getval(g_ceph_context, cmdmap, "prefix", prefix);
+  bool valid_cmd = cmd_getval(g_ceph_context, cmdmap, "prefix", prefix);
+
+  // check return value and prefix.
+  if (prefix.empty()) {
+    reply_command(op, -EINVAL, "empty command found", 0);
```

Comment:
please change the error message to something more obvious. E.g., "command prefix must not be empty".